src/server/conn_context.cc

```diff
@@ -134,7 +134,7 @@ vector<unsigned> ChangeSubscriptions(bool pattern, CmdArgList args, bool to_add,
   // removed.
   if (!to_add && conn_state.subscribe_info->IsEmpty()) {
     conn_state.subscribe_info.reset();
-    conn->force_dispatch = false;
+    conn->subscriptions--;
```

Comment:
```suggestion
    CHECK_GT(conn->subscriptions, 0);
    conn->subscriptions--;
```

Reply:
its unsigned

Reply:
Yeah, but you can still check `conn->subscriptions > 0` to not underflow